pyproject.toml

```
@@ -29,7 +29,7 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 etl = [
-    "variation-normalizer~= 0.12.1",
+    "ga4gh.va_spec @ git+https://github.com/ga4gh/va-spec-python.git@b9e803573e36b4f560e843459d8655b8d5692a7b",
```

Just made a release in Variation Normalizer 

```suggestion
    "variation-normalizer~= 0.14.0",
    "ga4gh.va_spec @ git+https://github.com/ga4gh/va-spec-python.git@b9e803573e36b4f560e843459d8655b8d5692a7b",
```